api/src/main/resources/client_api.yaml

```diff
@@ -124,64 +150,111 @@ definitions:
-          (In Java, this is represented as Map<String, Object>[]. In Python clients, this is a
-          list[object] where each object is a dictionary. In Typescript clients, this is an
-          Array<any> where each object is a dictionary.) Keys in the dictionaries will be the
-          columns selected in the field set provided in the request, and the values will
-          be the values of those columns.
+      name:
+        description: the name of the parameter
+        type: string
+      value:
+        description: A string value for this parameter.
+        type: string
+      values:
+        description: An array of string values for this parameter.
         type: array
         items:
-          type: object
-      nextPageToken:
-        description: >
-          Pagination token that can be used in a subsequent call to MaterializeCohortRequest to
-          retrieve more results. If not set, there are no more results to retrieve.
+          type: string
+      valueDate:
+        description: A date (yyyy-MM-dd) value for this parameter
+        type: string
+      valueDatetime:
+        description: A datetime (yyyy-MM-dd HH:mm:ss zzz) value for this parameter
         type: string
+      valueNumber:
+        description: A number value for this parameter.
+        type: number
+      valueNumbers:
+        description: An array of number values for this parameter
+        type: array
+        items:
+          type: number
 
-  FieldSet:
+  CohortAnnotationsRequest:
     type: object
-    description: >
-      A specification for fields to retrieve about participants in a cohort.
-      Exactly one of the properties below should be specified.
+    required:
+      - cohortName
     properties:
-      tableQuery:
+      cohortName:
         description: >
-          A query specifying how to pull data out of a single table. Either this or annotationQuery
-          should be set (not both.)
-        $ref: "#/definitions/TableQuery"
+         The name of a cohort that annotations should be retrieved for.
+        type: string
+      statusFilter:
+        description: >
+          An array of status values; participants with these statuses will be have their annotations
+          retrieved. Defaults to [NOT_REVIEWED, INCLUDED, NEEDS_FURTHER_REVIEW] --
+          everything but EXCLUDED.
+        type: array
+        items:
+          $ref: "#/definitions/CohortStatus"
+      cdrVersionName:
+        description: >
+          The name of a CDR version to use when retrieving annotations; if none is specified,
+          the CDR version currently associated with the workspace will be used
+        type: string
       annotationQuery:
         description: >
-          A query specifying how to retrieve annotation values created about participants in a
-          cohort during cohort review. Either this or tableQuery should be set (not both.)
+          Specification defining what annotations to retrieve.
```

Comment:
Would it make sense to add statusFilter from above to annotationQuery? Seems like they will be used together in implementation?

Reply:
See above.

Reply:
sorry for the dup question. For some reason when i submitted my review I couldn't see that question anymore and wasn't sure if I forgot to click button to add the comment to the review. Anyway looks good.